### PR TITLE
metrics: Fix optional module name api

### PR DESCRIPTION
**Pull request by juliangruber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -342,7 +342,7 @@ Returns `AsyncGenerator<ActivityEvent>`.
 
 Returns `Promise<MetricsEvent>`.
 
-### `Core#metrics.follow(module?: String, { signal?: AbortSignal })`
+### `Core#metrics.follow({ module?: String, signal?: AbortSignal })`
```
Comment (bajtos) [inline README.md:345]: Is it possible to omit both property args and call `follow` with no arguments at all?

```js
core.metrics.follow()
```

Maybe we can signal that option by adding `?` after the object.

```md
### `Core#metrics.follow({ module?: String, signal?: AbortSignal }?)`
```

Or perhaps:

```md
### `Core#metrics.follow({ module?: String, signal?: AbortSignal } = {})`
```

**Review by bajtos — APPROVED** [review]
👍🏻

File: README.md (hunk 1 above)
Comment (juliangruber) [reply README.md:345]: Yes it's possible. I had the `= {}` before, but it got too verbose for my taste. Will try again

**Comment by juliangruber** [comment]
https://github.com/filecoin-station/core/releases/tag/v8.0.0

**Pull request merged**